sampleinput.yaml

```diff
@@ -0,0 +1,14 @@
+creds:
```

Comment:
To follow up on my comments further down in this file, I think you can combine the git and creds sections like this:

```yaml
service-1:
  type: gitlab
  host: server-1.fake.url.com
  token: abc123
  repos:
    - user_name/repo_name
    - user_name
    - group_name/repo_name
    - group_name
service-2:
  type: gitlab
  host: server-1.fake.url.com
  token: abc123
  repos:
    - user_name/repo_name
    - user_name
    - group_name/repo_name
    - group_name
github:
  type: github
  token: abc123
  repos:
    - user_name/repo_name
    - user_name
    - org_name/repo_name
    - org_name
```

What do you think? Am I missing something that may prevent this?